internal/svcmanager/svcmanager.go

```diff
@@ -42,12 +43,12 @@ func (m *Manager) Start() error {
 	return nil
 }
 
-func (m *Manager) Wait() error {
+func (m *Manager) Wait(ping Pinger) error {
```

Comment:
This is super awkward. If we have a recursive dependency (manager needs model needs manager ..) let's take the model out of the equation. For a simple ping to see if the server is responding we don't need a model, we need an endpoint that returns a very basic status code. Just do a simple http request and check the result.

Reply:
Just to note, this is outdated.

Reply:
@Naatan You must have been reviewing an outdated changeset.